### PR TITLE
[MIG] report_substitute: Migration to 16.0

### PR DESCRIPTION
**Pull request by matiasperalta1**

### CODE REVIEW

**Review by amkarthik — APPROVED** [review]

**Review by atchuthan — APPROVED** [review]
Functional QA :+1:

**Comment by HviorForgeFlow** [comment]
I would migrate it directly from v13 to v16 if it your requirement. Other contributors that may need it in previous versions will backport it.

File: report_substitute/demo/action_report.xml [hunk 1]
```diff
@@ -3,7 +3,7 @@
     <template id="substitution_report">
         <div class="page">Substitution Report</div>
     </template>
-    <report
+    <record
```
Comment (HviorForgeFlow) [inline report_substitute/demo/action_report.xml]: That's not right. Here you have an example of how to transform report tags to record: I would migrate it directly from v13 to v16 if it your requirement. Other contributes that may need it in previous versions will backport it.

I also recommend you to read those wikis when migrating: 
https://github.com/OCA/maintainer-tools/wiki#migration

**Review by HviorForgeFlow — CHANGES_REQUESTED** [review]

**Review by HviorForgeFlow — APPROVED** [review]

**Pull request merged**